### ISSUE TITLE
Mesh network code

### ISSUE THREAD

**Issue by pavankumar8279** [contiki-os/contiki#2439]
Hi 
I am trying to create mesh network using cc2650 launchpad five in number making four as nodes
and one board as gateway I am planning to use udp-client.c/udp-server.c example to send and receive data 
so how can I add mesh network features to it so that data will be received efficiently using ipv6 stack?
or Is there any other example code in Contiki except "rime"
or any functions available to use it for creating a mesh network

Thank you and Regards
Pavan

**Comment by ankit2002** [comment]
Hey Pavan, Have you got any solution for this?

**Comment by pavankumar8279** [comment]
Yes, the udp-client.c and udp-server.c code are itself creates a mesh network, that I didn't knew at that time and I tested on cc2650 launchpad

**Comment by gnamyy** [comment]
Hi all,
sorry for intrusion
which is the command to compile the rpl-udp?
